### DISCUSSION TITLE
Change event colors

### DISCUSSION THREAD

**Discussion by Creasiion** [niccokunzmann/open-web-calendar#375] [Q&A]
I was just wondering if there was a customizable feature to display our event colors. All of my events are under the same Gcalendar, but I changed their event color to organize them (i.e. red for due dates, blue for work).

**Comment by niccokunzmann** [comment]
Hi, could you attach the ICS file(s) that you would like to display? Then, it is easier to see if that information is also available.

**Reply by niccokunzmann** [reply]
This is discussed here now: https://github.com/niccokunzmann/open-web-calendar/issues/376
